chart/values.schema.json

```diff
@@ -100,6 +100,34 @@
                 }
             ]
         },
+        "securityContexts": {
+            "description": "Security context definition. The values in this parameter will be used when `securityContexts` is not defined for specific Pods/Container",
+            "type": "object",
+            "x-docsSection": "Kubernetes",
+            "properties": {
+                "pod": {
+                    "description": "Pod security context definition. The values in this parameter will be used when `securityContexts` is not defined for specific Pods",
+                    "type": "object",
+                    "$ref": "#/definitions/io.k8s.api.core.v1.PodSecurityContext",
+                    "default": {},
+                    "x-docsSection": "Kubernetes",
+                    "examples": [
+                        {
+                            "runAsUser": 50000,
+                            "runAsGroup": 0,
+                            "fsGroup": 0
+                        }
+                    ]
+                },
+                "container": {
+                    "description": "Container security context definition. The values in this parameter will be used when `securityContexts` is not defined for specific containers",
+                    "type": "object",
+                    "$ref": "#/definitions/io.k8s.api.core.v1.SecurityContext",
+                    "default": {},
```

Comment:
There is a default though?

Reply:
removed the default to ensure backwards compatibility.
also I could not figure out how to reference it in https://github.com/apache/airflow/blob/1cc1d0a93e7445e0179178d370bfdc6dd8cab250/chart/values.schema.json#L126

Reply:
actually I moved it to the helper: https://github.com/apache/airflow/blob/85ec1fede2848ae8dc45dca8d0f933c9e400c9f7/chart/templates/_helpers.yaml#L786-L814